tests/framework/integration/testing.go

```diff
@@ -29,6 +30,7 @@ import (
 	clientv3 "go.etcd.io/etcd/client/v3"
 	"go.etcd.io/etcd/server/v3/embed"
 	gofail "go.etcd.io/gofail/runtime"
```

Comment:
Looks like a minor issue here causing static analysis to fail, suggested fix below:

```diff
  	"go.etcd.io/etcd/client/pkg/v3/verify"
 	clientv3 "go.etcd.io/etcd/client/v3"
 	"go.etcd.io/etcd/server/v3/embed"
-	gofail "go.etcd.io/gofail/runtime"
 	"go.etcd.io/etcd/tests/v3/framework/testutils"
+	gofail "go.etcd.io/gofail/runtime"
```